wordpress-mobile/WordPress-iOS
# Update to cocoa pods 1.3.1

This PR updates the cocoapod version to 1.3.1

To test:
 - Try to build the app from scratch using the version 1.3.1 
 - Run the app on simulator and device
 - Try to make an archive and check if all is good.
